.changeset/pattern-constraint-regex.md

```
@@ -0,0 +1,41 @@
+---
+'@conform-to/dom': minor
+'@conform-to/react': minor
+'@conform-to/valibot': minor
+'@conform-to/zod': minor
+---
+
+Mapped regex constraints to `pattern`, enabling advanced client-side validation.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)